test/test_collection.py

```diff
@@ -182,6 +182,14 @@ def test_estimated_document_count(self):
         last_cmd_payload = self.explain.last_cmd_payload
         self._compare_command_dicts(last_cmd_payload, last_logger_payload)
 
+    def test_cli_tool(self):
+        import subprocess
```

Comment:
No reason to have this be a local import - please put it at the top of the file.

Reply:
Done.